kernel/src/main/javacc/Outer.jj

```diff
@@ -10,8 +10,6 @@ the behavior of the previous system.
 
 /* DISCUSSION ITEMS:
 
-Is one of "priorities" or "priority" depricated/preferred? If so, should the other trigger a warning?
```

Comment:
😄

Reply:
Looks like that comment is 11 years old too!